test/e2e/e2e_suite_test.go

```diff
@@ -176,6 +173,26 @@ func applyManifests(kindCluster kind.Cluster, manifests *string) {
 	kindCluster.ApplyYAML(*manifests)
 }
 
+func deployCAPIComponents(kindCluster kind.Cluster) {
+	fmt.Fprintf(GinkgoWriter, "Generating CAPI manifests\n")
+
+	// Build the manifests using kustomize
+	capiManifests, err := exec.Command(*kustomizeBinary, "build", "https://github.com/kubernetes-sigs/cluster-api//config/default").Output()
+	if err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			fmt.Fprintf(GinkgoWriter, "Error: %s\n", string(exitError.Stderr))
+		}
+	}
+	Expect(err).NotTo(HaveOccurred())
+
+	// write out the manifests
+	manifestFile := path.Join(suiteTmpDir, "cluster-api-components.yaml")
+	Expect(ioutil.WriteFile(manifestFile, capiManifests, 0644)).To(Succeed())
+
+	// apply generated manifests
+	applyManifests(kindCluster, &manifestFile)
```

Comment:
Follow-up: change `applyManifests` so it takes in a `string`, not a `*string`?

Reply:
hmm, now that you mention it this should probably just call `kindCluster.ApplyYAML` instead :joy: